contributing.md

```diff
@@ -11,6 +11,16 @@ Regarding Pybind, Here is a good [introduction](https://www.youtube.com/watch?v=
 
 ## Setting up your environment
 
+### Git Submodules
+
+Currently we are using git submodules to allow editors like VSCode to locate source code, this should be refactored later. Don't forget to init and update the submodules or you will get an error on building.
+
+```
+$ git submodule update --init --recursive
```

Comment:
We already have the script for same at https://github.com/OpenMined/PyDP/blob/dev/ext_source_setup. 
Is this complexity needed?

Reply:
Hey, yeah good point sorry, I didn't actually see that, but perhaps the README.md wasn't clear to me, I have fixed a few issues and updated the README to explain the three different shell scripts and cleaned up some consistency issues etc. Not trying to add unnecessary noise to the git history so feel free to reject, but I did miss the submodule thing initially and wasted a bit of time debugging bazel errors.